dao/src/test/java/org/thingsboard/server/dao/service/BaseTenantServiceTest.java

```diff
@@ -275,4 +323,374 @@ public void testSaveTenantWithIsolatedProfileInMonolithSetup() {
         tenant.setTenantProfileId(isolatedTenantProfile.getId());
         tenantService.saveTenant(tenant);
     }
+
+    @Test
+    public void testGettingTenantAddItToCache() {
+        Tenant tenant = new Tenant();
+        tenant.setTitle("My tenant");
+        Tenant savedTenant = tenantService.saveTenant(tenant);
+
+        reset(tenantDao);
```

Comment:
Please, use the full notation Mockito.reset() as it is a rarely used method. Other static imports are fine